### PR TITLE
Issue #6207: Add Xpath Regression Test for InnerTypeLastTest

### PR DESCRIPTION
**Pull request by TanayMorakhia**
part of #6207

### CODE REVIEW

**Review by romani — CHANGES_REQUESTED** [review]
Good, please add one more test for inner type in inner type.
We asking to have 3 tests for each Check.

**Comment by TanayMorakhia** [comment]
Done.

File: src/it/resources/org/checkstyle/suppressionxpathfilter/innertypelast/SuppressionXpathRegressionInnerTypeLastThree.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.checkstyle.suppressionxpathfilter.innertypelast;
+
+public class SuppressionXpathRegressionInnerTypeLastThree {
+
+    static {} // OK
+
+    interface Inner {
+    }
+
+    public SuppressionXpathRegressionInnerTypeLastThree() { // warn
```
Comment (romani) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/innertypelast/SuppressionXpathRegressionInnerTypeLastThree.java:10]: Two and Three are almost identical.

I meant:
```
public class SuppressionXpathRegressionInnerTypeLastTwo {

    static {}; // OK
    
    public void innerMethod() {} 

class Inner {



};

}
```

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/it/resources/org/checkstyle/suppressionxpathfilter/innertypelast/SuppressionXpathRegressionInnerTypeLastThree.java (hunk 1 above)
Comment (TanayMorakhia) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/innertypelast/SuppressionXpathRegressionInnerTypeLastThree.java:10]: done.

**Review by romani — APPROVED** [review]

**Pull request merged**